framework/src/transfers/MultiAppInterpolationTransfer.C

```diff
@@ -410,7 +410,9 @@ MultiAppInterpolationTransfer::execute()
 {
   _console << "Beginning InterpolationTransfer " << name() << std::endl;
 
-  const FEProblemBase & fe_problem = _multi_app->problemBase();
+  // CHECK THIS
```

Comment:
?

Reply:
oh that was from early on I ll delete it